cvat-ui/src/components/annotation-page/top-bar/player-navigation.tsx

```diff
@@ -85,18 +91,52 @@ function PlayerNavigation(props: Props): JSX.Element {
             });
         }
     }, [playing, frameNumber]);
+
+    const subKeyMap = {
+        DELETE_FRAME: keyMap.DELETE_FRAME,
+        FOCUS_INPUT_FRAME: keyMap.FOCUS_INPUT_FRAME,
+    };
+
+    const handlers = {
+        DELETE_FRAME: (event: KeyboardEvent | undefined) => {
+            event?.preventDefault();
+            onDeleteFrame();
+        },
+        FOCUS_INPUT_FRAME: (event: KeyboardEvent | undefined) => {
+            event?.preventDefault();
+            if (inputFrameRef.current) {
+                inputFrameRef.current.focus();
+            }
+        },
+    };
+
+    const disabledStyle: CSSProperties = {
+        pointerEvents: 'none',
+        opacity: 0.5,
+    };
+
     const deleteFrameIcon = !frameDeleted ? (
         <CVATTooltip title={`Delete the frame ${deleteFrameShortcut}`}>
-            <DeleteOutlined className='cvat-player-delete-frame' onClick={showDeleteFrameDialog} />
+            <DeleteOutlined
+                style={workspace === Workspace.SINGLE_SHAPE ? disabledStyle : {}}
```

Comment:
Lets create a separate variable instead of checking `workspace === Workspace.SINGLE_SHAPE` every time

Reply:
Done